### ISSUE TITLE
Theme support?

### ISSUE THREAD

**Issue by iUltimateLP** [christofmuc/KnobKraft-orm#147]
Hey there, great stuff! I haven't digged into the source code much yet, but I'm wondering if it's possible to implement theming? I'll fork this later and try to work myself into the JUCE framework, but I'd generally like to be able to make changes for the different colors, e.g. allowing a more flat theme, etc.

Let me know whether that's something imaginable or something rather annoying with the current state of the project. Cheers!

**Comment by christofmuc** [comment]
@iUltimateLP Actually, I tried to make KnobKraft adhere to the JUCE rules for "Look and Feel", which is their theming. I remember at the beginning before I decided on these colours, I had the possibility to switch between dark and light mode. There will probably be a few problems here and there but those should be correctable. 

Search for OrmLookAndFeel and setColourScheme. Currently, the code is commented out and I just use a standard JUCE colour scheme.
